roles/wordpress-setup/tasks/self-signed-certificate.yml

```diff
@@ -1,10 +1,19 @@
 ---
 - name: Generate self-signed certificates
-  shell: >
-    openssl req -subj "/CN={{ item.value.site_hosts[0].canonical }}" -new
-    -newkey rsa:2048 -days 3650 -nodes -x509 -sha256
-    -keyout {{ item.key }}.key -out {{ item.key }}.cert
+  shell: "openssl req -subj \"/CN={{ item.value.site_hosts[0].canonical }}\" -new \
```

Comment:
The change of command style was mandatory for a few reasons, first one is that the brackets characters (l.7-10) have to be escaped or ansible throws a syntax error.

The original `Block` scalar style using a `>` removes the newlines within the string, thus not allowing the `config` openssl section to work properly.

The choice of the `Flow` scalar style using a `"` (double-quote) is the only style in yaml multi-line string allowing the use of `inline newlines` (`\n`) and `spaceless newlines` (trailing `\`) (c.f. http://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines#answer-21699210).

I could have also used the `Flow` literal style using a `|`, but that would have forced all the dashed parameters of the 
 openssl command to be written on the same line since a spaceless newline (trailing `\`) can't be added in this mode.

In the chosen style, the double quotes `"` and backslash `\` have to be escaped though.